### PR TITLE
Hackerrank3: java collection framework

### PR DESCRIPTION
**Pull request by Ghumber**

### CODE REVIEW

File: Humbertino_Avilez/Day3/Hackerrank/JavaSort.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+			elements--;
+		}
+		Collections.sort(studentList,Collections.reverseOrder());
+		for(Student st: studentList){
+			System.out.println(st.getFname());
+		}
+	}
+}
+
+class Student implements Comparable<Object> {
```
Comment (globant-bootcamp) [inline Humbertino_Avilez/Day3/Hackerrank/JavaSort.java:29]: extract this "class" in to a different file

File: Humbertino_Avilez/Day3/Hackerrank/JavaSort.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	private int id;
+	private String fname;
```
Comment (globant-bootcamp) [inline Humbertino_Avilez/Day3/Hackerrank/JavaSort.java:31]: fname?

File: Humbertino_Avilez/Day3/Hackerrank/JavaSort.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	private double cgpa;
```
Comment (globant-bootcamp) [inline Humbertino_Avilez/Day3/Hackerrank/JavaSort.java:32]: cgpa?

File: Humbertino_Avilez/Day3/Hackerrank/JavaSort.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	public String getFname() {
+		return fname;
+	}
+
+	public double getCgpa() {
+		return cgpa;
+	}
+
+	@Override
+	public int compareTo(Object o) {
```
Comment (globant-bootcamp) [inline Humbertino_Avilez/Day3/Hackerrank/JavaSort.java:53]: o?

File: Humbertino_Avilez/Day3/Hackerrank/JavaArraylist.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+public class JavaArraylist {
+	static List<List<Integer>> mainList = new ArrayList<List<Integer>>();
+	static Scanner sc = new Scanner(System.in);
+
+	public static void main(String[] args) {
+		int lines = sc.nextInt();
+		sc.nextLine();
+		while (lines > 0) {
+			List<Integer> temporaryList = new ArrayList<>();
+			String[] elements = sc.nextLine().split(" ");
```
Comment (globant-bootcamp) [inline Humbertino_Avilez/Day3/Hackerrank/JavaArraylist.java:16]: Remplace all the hardcode String by static final variables

**Review by globant-bootcamp — CHANGES_REQUESTED** [review]

**Review by globant-bootcamp — APPROVED** [review]

**Review by luisod78 — APPROVED** [review]

**Pull request merged**